src/com/facebook/buck/maven/Resolver.java

```diff
@@ -92,73 +114,133 @@
   private final Path buckRepoRoot;
   private final Path buckThirdPartyRelativePath;
   private final LocalRepository localRepo;
-  private final ImmutableList<RemoteRepository> repos;
   private final ServiceLocator locator;
   private final VersionScheme versionScheme = new GenericVersionScheme();
+  private final ModelBuilder modelBuilder;
+
+  private ImmutableList<Dependency> initialRequest;
```

Comment:
Maybe its better to rename this ivar. `initialRequest` and `List` types don't sound together well. Maybe something like `initialRequestDeps` or something like that?